kentikapi/client.go

```diff
@@ -200,3 +201,21 @@ func makeRetryInterceptor(c Config) grpc.UnaryClientInterceptor {
 		grpcretry.WithMax(*c.RetryCfg.MaxAttempts+1),
 	)
 }
+
+func makeLoggerInterceptor(c Config) grpc.UnaryClientInterceptor {
+	return func(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn,
+		invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
+		c.LogPayloads = true
+		if c.LogPayloads {
+			log.Printf("Kentik API request - %s - %+v\n", method, req)
+		}
+		err := invoker(ctx, method, req, reply, cc, opts...)
+		if err != nil {
```

Comment:
No. Logging response should not be skipped when underlying code returns error. We need that information in logs!